src/BizHawk.Client.Common/movie/interfaces/IMovie.cs

```diff
@@ -46,12 +46,12 @@ public interface IMovie
 
 		/// <summary>
 		/// Gets the total number of frames that count towards the completion time of the movie
-		/// Possibly (but unlikely different from InputLogLength (could be infinity, or maybe an implementation automatically discounts empty frames at the end of a movie, etc)
 		/// </summary>
-		double FrameCount { get; }

```

Comment:
So I pondered InputLogLength vs FrameCount and wanted to document my thinking.  I considered dropping one of them since they are the same. But we could anticipate they would not if we had a notion of "subframe" input.  

Normally I do not like prebuilding since in my experience it is often done wrong (you never fully anticipate future needs).  However, the work has already been done here and the calling code already has sorted out whether it cares about frame count or actual log length so I kept it in there.

Also, a double is arguably a good value for FrameCount if subframes are a thing.  But I decided that's nonsense and an implementation should define 100.5 frames as 101 frames.  Or at the very least, it deserves a future nerd fight